phpunit.xml.dist

```diff
@@ -138,6 +138,7 @@
 		<directory>tests/Backend/CommonPart2</directory>
 		<exclude>tests/Backend/Snowflake/CloneIntoWorkspaceTest.php</exclude>
 		<exclude>tests/Backend/Snowflake/CopyIntoWorkspaceTest.php</exclude>
+		<exclude>tests/Backend/Snowflake/TypedTableWorkspacesLoadTest.php</exclude>
```

Comment:
proc? proc se to ma testovat v gcs a abs, ale ne na s3?

Reply:
ok, to je mi jasne, tak to pak dejme jen do jednoho, takto to bude v gcs i abs. A treba i s komentarem :)

Reply:
paratest-backend-snowflake-part-1

Reply:
paratest-backend-snowflake-part-1 sa spůšťá v CI enom jednů a řekl bych že pro S3

Reply:
no `<directory>tests/Backend/Snowflake</directory>` je ve suitach s3, abs i gcs. Tzn pokud se nic nestane, tak se `TypedTableWorkspacesLoadTest` pusti ve vsech trech. Ty to excludujes jen v s3, takze se to porad pousti v gcs i abs. Tak ja rikam - excludneme to jeste (treba) v abs, bude se to tim padem poustet jen v gcs -> pusti se to jen 1 a vsichni jsme happy

Reply:
jo už mě to docvaklo :D Opraveno, dík